### ISSUE TITLE
use 1 instance of HeaterCoolerNode

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/gas-properties#160]
"Pressure ↕T" mode for "Hold Constant" animates the heat/cool bucket to indicate how temperature `T` is adjusted to keep pressure `P` constant.  The animation is handled by a dedicated instance of `AnimatedHeaterCoolerNode` that is visible only in this mode.  An instance of `GasPropertiesHeaterCoolerNode` is visible in all other modes.  

Having 2 instances of the bucket may be confusing for PhET-iO.   So consider folding the implementation of `AnimatedHeaterCoolerNode` into `GasPropertiesHeaterCoolerNode`, so that 1 instance handles all modes.

This is deferred until after 1.0.0 publication, when PhET-iO instrumentation resumes.

**Comment by pixelzoom** [comment]
Some badness in `AnimatedHeaterCoolerNode`:

```js
      // @private {Animation|null}
      this.animation = null;

      // stops the animation at whatever stage it's in
      const stopAnimation = () => {
        if ( this.upAnimation ) {
          this.upAnimation.stop();
          this.upAnimation = null;
        }
      };
```

`this.upAnimation` does not exist, should be `this.animation`.  But be careful about just changing it; it results in jerky behavior.    This needs to be removed in the animation code on line 108:

```js
              // stop any animation that is in progress
              stopAnimation();
```

**Comment by pixelzoom** [comment]
Addressing this issue is the preferred way of addressing #164.

**Comment by pixelzoom** [comment]
This will be addressed and regression tested in #164.  Closing.
